controller/src/test/java/org/dswarm/controller/resources/resource/test/DataModelsResourceTest2.java

```diff
@@ -214,6 +219,56 @@ public void testCSVDataUpdate2() throws Exception {
 		Assert.assertTrue(data.isPresent());
 		Assert.assertNotNull(assoziativeJsonArray);
 
+		final String expectedResult = DMPPersistenceUtil.getResourceAsString("dd-762.expected.delta-result.json");
+		final String actualResult = objectMapper.writeValueAsString(assoziativeJsonArray);
+
+		// TODO: do proper comparison
+		Assert.assertEquals(expectedResult.length(), actualResult.length());
```

Comment:
no, or when we like to apply it's a bit more complicated here. the records uris will always be generated and differ then from the execepted result. so one always need to replace them before comparing the JSON with JsonAssert.